docs/admin/install/docker.rst

```diff
@@ -2306,60 +2306,69 @@ LibreTranslate Integration
 translation service that can be self-hosted. Integrating it with Weblate provides
 offline machine translation capabilities without relying on external services.
 
-To add LibreTranslate to your Weblate setup, extend your :file:`docker-compose.yml`
-or create a :file:`docker-compose.override.yml` with the following configuration:
+The setup is surprisingly simple - you can add LibreTranslate to your Weblate deployment
+by creating a :file:`docker-compose.override.yml` file with the LibreTranslate service.
+Since it runs within the Docker network, it's only accessible to Weblate and not exposed
+to the public internet.
+
+Basic setup using :file:`docker-compose.override.yml`:
 
 .. code-block:: yaml
 
-   version: '3'
    services:
      libretranslate:
+       container_name: libretranslate
        image: libretranslate/libretranslate:latest
-       restart: unless-stopped
-       ports:
-         - "5000:5000"
+       command: --disable-web-ui
        environment:
-         LT_HOST: 0.0.0.0
-         LT_PORT: 5000
-         # Uncomment and set API key for production use
-         # LT_API_KEYS: true
-         # LT_API_KEYS_DB_PATH: /app/db/api_keys.db
-         # LT_REQUIRE_API_KEY_ORIGIN: true
+         LT_UPDATE_MODELS: true
        volumes:
-         - libretranslate-data:/app/db
-       healthcheck:
-         test: ["CMD-SHELL", "curl -f http://localhost:5000/health || exit 1"]
-         interval: 30s
-         timeout: 10s
-         retries: 3
-       # Uncomment to limit resources
-       # deploy:
-       #   resources:
-       #     limits:
-       #       memory: 2G
-       #       cpus: '1.0'
-
-   volumes:
-     libretranslate-data:
-
-After starting the services, configure LibreTranslate in Weblate:
+         - ./libretranslate_db:/app/db
+         - ./libretranslate_data:/root/.local:rw
```

Comment:
The volumes seems wrong - the non-cuda container uses regular user, not root. https://github.com/LibreTranslate/LibreTranslate/blob/main/docker-compose.yml has:

```
- libretranslate_models:/home/libretranslate/.local:rw
- ```